openvdb/tree/RootNode.h

```diff
@@ -51,7 +51,8 @@
 #include <map>
 #include <set>
 #include <sstream>
-#include <deque>
+#include <vector>
+#include <memory> // for std::make_unique
```

Comment:
Think this header is not needed anymore? (You're no longer using a std::unique_ptr?)